functions/cloudflare_b2_proxy.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Cloudflare Workers script to proxy requests to Backblaze B2 with 
```

Comment:
I am a bit confuse, why is this a Cloudflare workers script? I don't see anywhere in the code that this script talks to Cloudflare. Or am I missing something?

Or maybe you means that this script is written and meant to be hosted in Cloudflare? 
If so, I would encourage to change the comment as one day we could decided to host it somewhere else and i believe it will still work.